src/components/Pill.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+type Props = {
+  color?: 'danger' | 'success' | 'warning' | 'primary' | 'secondary' | 'neutral';
```

Comment:
Can't we use an enum here?

Reply:
This would mean you have to write something like `color={Color.Danger}` if you want to set the color.
I don't want to force devs to import an extra type to use the component.
But in the first place I did this for consistency compared to the other components.